Makefile

```diff
@@ -64,33 +48,30 @@ clean: 												## Cleanup temporary build artifacts
 destroy: 											## Destroy the virtual environment
 	@rm -rf .venv
 
-.PHONY: refresh-lockfiles
-refresh-lockfiles:                                 ## Sync lockfiles with requirements files.
-	pdm update --update-reuse --group :all
 
 .PHONY: lock
 lock:                                             ## Rebuild lockfiles from scratch, updating all dependencies
-	pdm update --update-eager --group :all
+	uv lock
 
 # =============================================================================
 # Tests, Linting, Coverage
 # =============================================================================
 .PHONY: mypy
 mypy:                                               ## Run mypy
 	@echo "=> Running mypy"
-	@$(PDM) run dmypy run
+	@uv run dmypy run
```

Comment:
should our lint/format/etc. use `no-sync`?

Reply:
e.g., like @DanCardin's https://github.com/DanCardin/cappa/blob/main/Makefile https://github.com/DanCardin/cappa/blob/main/.github/workflows/test.yml#L61

Reply:
Hm. Idk. I haven't really settled on an opinion. maybe just leave it like this for now and see how it works out?

Reply:
In the workflows we set `UV_LOCKED`, so we get a warning if something's not up to date